docs/topics/images.rst

```diff
@@ -1,76 +1,94 @@
 .. _topics-images:
 
-=======================
-Downloading Item Images
-=======================
+===========================================
+Downloading and processing files and images
+===========================================
 
 .. currentmodule:: scrapy.contrib.pipeline.images
 
-Scrapy provides an :doc:`item pipeline </topics/item-pipeline>` for downloading
-images attached to a particular item, for example, when you scrape products and
-also want to download their images locally.
+Scrapy provides reusable :doc:`item pipelines </topics/item-pipeline>` for
+downloading fies attached to a particular item (for example, when you scrape
+products and also want to download their images locally). These pipelines share
+a bit of functionality and structure (we refer to them as media pipelines), but
+typically you'll either use the Files Pipeline or the Images Pipeline.
 
-This pipeline, called the Images Pipeline and implemented in the
-:class:`ImagesPipeline` class, provides a convenient way for
-downloading and storing images locally with some additional features:
+Both pipelines implement these features:
+
+* Avoid re-downloading media that was downloaded recently
+* Specifying where to store the files (filesystem directory, Amazon S3 bucket)
```

Comment:
I think of it like this: media is more about the content and file is more about the "container/artifact".

So, we avoid re-downloading media because we're assuming the same URL would give us the same document/image/video.
We specify where to store the files because for a given media (image), we may store two files (original + thumbnail).

I'm not sure if this makes sense to someone other than me, though. :D